### ISSUE TITLE
root post ist not displayed?

### ISSUE THREAD

**Issue by cryptix** [ssbc/ssb-viewer#1]
See the link in https://github.com/ssbc/scuttlebot/issues/491

was also reported and examplified in `%U6MQT5dlMxdzA3NA4YbD/kiPUpshzS8kdTbTXvs1eik=.sha256`:

> Weird, [this thread](%CyLw6EMH54VV6CuYgYYWK0DXSotYg8abXmFCYXOKj1w=.sha256) misses the OP in ssb-viewer: https://viewer.scuttlebot.io/%25CyLw6EMH54VV6CuYgYYWK0DXSotYg8abXmFCYXOKj1w%3D.sha256

**Comment by arj03** [comment]
Yeah, not sure why mainline has this bug or if it is only a bug in viewer.scuttlebot.io, but my pub running the calm branch displays the message fine:

http://between-two-worlds.dk:8807/%25Wou59mdQR37z0qDUDZMpAzvS%2FNdnZrqT5iC9SHYzOAA%3D.sha256

There is a open PR for merging the calm branch, but I don't think @clehner has had time to look at it yet. To his defence no wonder since a lot of changes rolled into one 😏

**Comment by ymarcus93** [comment]
@arj03 Just an FYI, your viewer doesn't respect the `--no-publicWebHosting` option.

**Comment by keks** [comment]
I literally just wanted to open this issue because of this one: https://viewer.scuttlebot.io/%25MkwJfB73tEiAofzB4yc4gZ6DOXz%2BRdOpDFm6S7PvJzU%3D.sha256

Synchonicity....

**Comment by arj03** [comment]
@ymarcus93 where are you seeing that? If you do to http://between-two-worlds.dk:8807/user-feed/@6CAxOI3f+LUOVrbAl0IemqiS7ATpQvr9Mdw9LC4+Uv0=.ed25519 you can see a bunch of "User has chosen not to be hosted publicly".

**Comment by ymarcus93** [comment]
Example: My own feed. View it on your viewer: http://between-two-worlds.dk:8807/@EzSBpjJkrXjUnjbOQCppGWnpgkhHnSkSBEUq8kTI6E8=.ed25519

On my viewer: https://viewer.diefreien.club/@EzSBpjJkrXjUnjbOQCppGWnpgkhHnSkSBEUq8kTI6E8=.ed25519

I think it might not be a bug with your calm branch, because I noticed when I switched off public hosting for my feed, I had to restart my server before the change showed up.

**Comment by arj03** [comment]
Whooops yeah that's a bug, good catch! I'll fix that.

**Comment by dominictarr** [comment]
sorry I think I made this one in 08321fdd28495d2cb08d4f7256ed2f6aab925315
